c3d/src/C3d.cpp

```diff
@@ -81,6 +92,79 @@ class C3dSimpleSpelling: public ParsedAttrInfo {
 private:
   Spelling S[1];
 
+protected:
+  static std::vector<c3d_var_info> c3d_scope;
```

Comment:
This might be just my C++ illiteracy showing :). I'm using static as a way to share this vector across the many inherited instances, so that when any of them pushes to scope, the others "see" it. It's basically a glorified global variable. Is there a better pattern here?